packages/turndown/src/commonmark-rules.js

```diff
@@ -266,6 +281,9 @@ rules.fencedCodeBlock = {
 
     var fence = repeat(fenceChar, fenceSize)
 
+    // remove code block leading and trailing newlines
+    code = code.replace(/^( *\t*\n)+/, '').replace(/( *\t*\n)+( *\t*)*$/, '')
```

Comment:
```suggestion
    // remove code block leading and trailing empty lines
    code = code.replace(/^( *\t*\n)+/, '').trimEnd()
```
Is `.replace(/( *\t*\n)+( *\t*)*$/, '')` equivalent to `.trimEnd()` in this case? If so, it could make sense to replace this.

Reply:
Yes, they are equal.